helpdesk_mgmt_ticket_related_users/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,41 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models, _, Command
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = 'helpdesk.ticket'
+
+    related_user_ids = fields.Many2many(
+        'res.users',
+        'helpdesk_ticket_related_user_rel',
+        'ticket_id', 'user_id',
+        string='Related Users',
+        domain="team_id and ['&','&', ('id','!=', user_id), ('share','=',False), ('id','in', user_ids)] or [('id','=',False)]",
+        help='Users involved but not assigned',
+        compute="_compute_related_user_ids",
+        readonly=False,
+        store=True
```

Comment:
```suggestion
        store=True
        tracking=True
```
Could you try using this? So when a new user is added to `related_user_ids`, on chatter we get something like this:
<img width="650" height="228" alt="imagen" src="https://github.com/user-attachments/assets/3467334c-7802-46fc-be47-0593da8f1db8" />
Could you provide an image with what you get to see if it's ok or too much information.

Reply:
<img width="505" height="266" alt="imagen" src="https://github.com/user-attachments/assets/466caa5c-2e12-4de1-8834-4bfca65127b1" />

Even when you change only one name, it posts all the related users. Maybe that’s too much.

Reply:
Ok, keep this aside, and we'll wait to see what the others have to say about it. Thanks!